master/docs/relnotes/index.rst

```diff
@@ -10,6 +10,161 @@ Release Notes
 
 .. towncrier release notes start
 
+Buildbot ``1.5.1-dev117`` ( ``2018-11-07`` )
```

Comment:
Should this be here? Seems like it shouldn't be filled out until the release happens.

Reply:
Fixed now.